apps/studio/data/api-keys/api-keys-query.ts

```diff
@@ -0,0 +1,81 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { get, handleError } from 'data/fetchers'
+import { ResponseError } from 'types'
+
+import { components } from 'api-types'
+import { apiKeysKeys } from './keys'
+
+type LegacyKeys = {
```

Comment:
these are okay for now - but ideally we grab them from the API types

Reply:
ideally, but we can't right now